docs/about/index.rst

```diff
@@ -8,6 +8,29 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
+The toolkit aspires to include:
+
+- Fairness metrics and unfairness mitigation algorithms
+- Interactive dashboard for fairness assessment
+- Educational materials (comprehensive user guide, detailed case studies,
+  white papers, etc.)
+
+Development of Fairlearn is firmly grounded in the understanding that fairness
+in AI systems is a sociotechnical challenge.  Because there are many complex
+sources of unfairness&mdash;some societal and some technical&mdash;it is not
+possible to fully "debias" a system or to guarantee fairness.
+Our goal is to enable humans to assess fairness-related harms, review the
+impacts of different mitigation strategies and then make trade-offs
+appropriate to their scenario.
+
+Fairlearn is a community-driven open source project, to be shaped through
```

Comment:
I think it might be more accurate to say that: Fairlearn is currently a collaboration between MSR and Azure, with the intention to transition to a community-driven open source project with an open governance structure in the next year.  Or whatever feels comfortable.

As has been discussed in the last two developer calls, it's not really community-driven now and that probably won't change in the next month or two.  As one example, it understandably seems quite important for MS folks to control the UX and visual design in fairlearn, since it needs to be able to be re-used within Azure.  I can't imagine it's ever going to be an option to consider UI elements that are not based on Fabric/FluentUI branding and theming :)  This seems totally reasonable and understandable, and based on the note about UX in task list below, it seems to more accurately reflect the current reality.  Perhaps I am misinterpreting, but I think it's helpful to just state any constraints like this directly  :)

Reply:
This is a good point. I think that we are more than a collaboration between MSR and Azure, but it is true that MSR and Azure have the final word, so it is not correct to say that we are "community-driven". What do you think about the following: Let's remove the "community-driven" part and explain the current decision making process in Governance section (still to be written...). Once we pin down the Governance section, we might be able to put something here and link to Governance for more details.